src/System.Net.Http/src/System/Net/Http/Managed/ManagedHandler.cs

```diff
@@ -0,0 +1,307 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http
+{
+    internal sealed class ManagedHandler : HttpMessageHandler
```

Comment:
I hate to be nitty here, but I'm not sure "ManagedHandler" is the best name for this.  It's rather non-descriptive.  I wonder if "ManagedClientHandler" is better... thoughts?